spec/api/sul_bib/authorship_api_spec.rb

```diff
@@ -364,6 +364,15 @@ def check_response_error(code)
         it_behaves_like 'it creates new contributions and publications' # TODO: modifies existing contributions.
       end
 
+      context 'with allcaps or mixed case strings' do
+        let(:request_data) { sul_author_hash.merge(sul_pub_id: publication_with_contributions.id, visibility: 'PRIVATE', status: 'New', featured: true) }
+        it 'downcases appropriately' do
+          expect { http_request }.not_to change { existing_contrib }
+          contrib = publication_with_contributions.contributions.reload.last
+          expect(contrib.status).to eq 'new'
+          expect(contrib.visibility).to eq 'private'
```

Comment:
The previous commit introduced some new patterns for checking the changes to attributes, use them here?  It was something like: `expect {}.not_to change { [attr1, ... , attrN ] }`.  Maybe this spec is also wanting to check there are no changes to the `Contribution.count`?  It seems to be checking that the last contribution is the same before/after, like it's not sure if the last contrib _is_ the `existing_contrib`?

Reply:
Can the JSON body have a string value for the boolean params?  If it can or usually does, should the specs also test it?

Reply:
This test is only concerned with the downcasing via Grape.  So it is not particularly concerned with response (tested elsewhere).  The POST definition requires values for `:featured` and `status` (also tested elsewhere).